### ISSUE TITLE
 MONDO:0005258, “Autism spectrum disorder”

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#2505]
**Mondo term (ID Label)**

 MONDO:0005258, “Autism spectrum disorder”

**Bug/Typo/Error description**
ASD Lists "pervasive developmental disorders" as a synonym but is also a child of MONDO:0000594 "pervasive developmental disorder"

Also unsure of the best hierarchy structure. UMLS has ASD as the parent term C1510586 with PDD as a narrower concept (C0524528) 
HPO treats "ASD" and "PDD" as synonyms to "autistic behavior" which I think is further complicating how UMLS processes these concepts. https://hpo.jax.org/app/browse/term/HP:0000729

I could really use some insight from the ontology community and any ASD experts to help clarify these term relationships. Thanks! 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by nicolevasilevsky** [comment]
thanks @kanems - I'll try to get to this soon. :)  If it's high priority, please let me know.

**Comment by nicolevasilevsky** [comment]
@pnrobinson could I ask you for feedback on this?

**Comment by nicolevasilevsky** [comment]
Looks like the synonym 'pervasive developmental disorders' comes from NCIt. @kanems I can ask them about it too, if it's helpful. Let me know.

**Comment by pnrobinson** [comment]
The classification of this has changed, see the first paragraph here
https://www.autismspeaks.org/pervasive-developmental-disorder-pdd-nos
I would suggest reviewing the entire hierarchy here and making sure it corresponds to DSM5 -- we can do this on a future call.

**Comment by nicolevasilevsky** [comment]
Action item:
- [ ] merge MONDO:0005261 'pervasive developmental disorder - not otherwise specified' with MONDO:0005258  'autism spectrum disorder'
- [ ] try to get an eBook of DSM-5 and review the autism related diseases
